api/shape/maskedBy.js

```diff
@@ -0,0 +1,14 @@
+import Shape from './Shape.js';
+import { maskedBy as op } from '@jsxcad/geometry';
+
+export const maskedBy = Shape.registerMethod3(
```

Comment:
How does this work?

Reply:
A mask subtracts its own shape instead of the shape it masks.

So you can, e.g., have a drill bit masked by a cylinder -- if you join that, you'll get the drill. If you subtract that, you'll remove a cylindrical cavity.

Mostly useful for adding play around an object.

Reply:
Ah that's right. I remember this now :)